### PR TITLE
Adding gappa/examineheattree

### PR DESCRIPTION
**Pull request by erikrikarddaniel**
## PR checklist

This adds a module for the `gappa examine heat-tree` program call.

- [x] This comment contains a description of changes (with reason).
- [x] If you've fixed a bug or added code that should be tested, add tests!
- [x] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [x] If necessary, include test data in your PR.
- [x] Remove all TODO statements.
- [x] Emit the `versions.yml` file.
- [x] Follow the naming conventions.
- [x] Follow the parameters requirements.
- [x] Follow the input/output options guidelines.
- [x] Add a resource `label`
- [x] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - [x] `PROFILE=docker pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=singularity pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=conda pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`

### CODE REVIEW

File: modules/gappa/examineheattree/meta.yml [hunk 1]
```diff
@@ -0,0 +1,62 @@
+name: "gappa_examineheattree"
+description: colours a phylogeny with placement densities
+keywords:
+  - phylogeny
+  - phylogenetic placement
```
Comment (d4straub) [inline modules/gappa/examineheattree/meta.yml:5]:
```suggestion
  - phylogenetic placement
  - heattree
  - visualisation
```

File: tests/modules/gappa/examineheattree/test.yml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+      md5sum: e1cfdcb63f3a751677d0e78fb6b9230a
+    - path: output/gappa/test.tree.phyloxml
+      md5sum: ea35b07e4fd8d174fd6b2f204629cc4e
+    - path: output/gappa/test.tree.svg
+    - path: output/gappa/test.colours.txt
+      md5sum: 7704cd341c8693981253ba6d740c1061
+    - path: output/gappa/test.log
+    - path: output/gappa/versions.yml
+      contains:
+        - "0.8.0"
```
Comment (d4straub) [inline tests/modules/gappa/examineheattree/test.yml:19]: I am a little lost why the versions.yml should vary in content (md5sum)? If it only contains the version string as tested for here it shouldnt vary?!

**Review by d4straub — APPROVED** [review]
LGTM, small comments below.

File: tests/modules/gappa/examineheattree/test.yml (hunk 2 above)
Comment (erikrikarddaniel) [reply tests/modules/gappa/examineheattree/test.yml:19]: I agree. I got variation a while ago for this container and have since just repeated that.

**Pull request merged**